vpd-tool/include/vpd_tool.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <string>
+
+namespace vpd
+{
+/**
+ * @brief Class to support operations on VPD.
+ *
+ * The class provides API's to,
+ * Read keyword value from DBus/hardware.
+ * Update keyword value to DBus/hardware.
+ * Dump DBus object's critical information.
```

Comment:
can you please list the critical data that we dump currently as part of dumpObject .